kim_cnn/SST1.py

```diff
@@ -0,0 +1,15 @@
+from torchtext import data
+import os
+
+
+class SST1Dataset(data.TabularDataset):
+    dirname = 'data'
+    @classmethod
+    def splits(cls, text_field, label_field,
+               train='phrases.train.tsv', validation='dev.tsv', test='test.tsv'):
+        prefix_name = 'stsa.fine.'
+        path = './kim_cnn/data'
+        return super(SST1Dataset, cls).splits(
+            path, train=prefix_name + train, validation=prefix_name + validation, test=prefix_name + test,
+            format='TSV', fields=[('label', label_field), ('text', text_field)]
```

Comment:
This script could not support current version of torchtext, could you refer the current code in Kim CNN and adapt this script ?
- format='TSV' should be format='tsv'

Do you have the script to build the docker? Could we also put that into the repo?

Reply:
Many thanks for the review! Will reply the feedback later.

For the docker image source code:
https://github.com/HeliWang/pytorch-mxnet-jvm

The version of pytorch/torchtext I used:
  torch 0.3.1
  torchtext 0.2.3